docs/kratos/social-signin/06_ory.mdx

```diff
@@ -0,0 +1,128 @@
+---
```

Comment:
1. This document seems to be a new doc in the "integrating social sign-in providers" section of the docs. These docs weren't supposed to be in the scope of your PR, right? 
2. The doc is not accessible through the sidebar. 
3. I don't understand why you would add an Ory Network project as a social sign-in provider to an Ory Network project - could you explain?

Reply:
Oh crap yea that was not supposed to be in this PR. 
And yea there should be some explanation of the usecase of this.
But will be handled in a different PR.